src/LibraryManager.Vsix/Shared/LibraryCommandService.cs

```diff
@@ -265,7 +284,15 @@ public bool IsOperationInProgress
 
                 sw.Stop();
 
-                Logger.LogEventsSummary(new List<ILibraryOperationResult> { result }, OperationType.Uninstall, sw.Elapsed);
+                if (result.Errors.Any())
```

Comment:
manifest.Uninstall() validates the Library we are trying to uninstall. If there are no errors, files for this library are deleted. Otherwise, we return the validation error.

Do we want to validate the entire manifest when uninstalling one of the libraries?